### ISSUE TITLE
Wrong value in call to F.softmax

### ISSUE THREAD

**Issue by marioyc** [BY571/Munchausen-RL#2]
Should `F.softmax(Q_targets_next, dim=1)` be `F.softmax(Q_targets_next / entropy_tau, dim=1)` instead?

**Comment by BY571** [comment]
for DQN its only Q_targets_next:

![image](https://user-images.githubusercontent.com/29492081/93122120-3fa5a800-f6c6-11ea-93d9-4ef18eefa187.png)

but for IQN you are right :)

**Comment by marioyc** [comment]
Oh, I didn't notice that, seems to contradict equation 2, and it would also change the logsumexp calculations, given that these assume the q values are divided by `entropy_tau `

**Comment by marioyc** [comment]
Confirmed with the author that it is a typo, the values should be divided by `entropy_tau`.
Also there is a TF implementation here: https://github.com/google-research/google-research/tree/master/munchausen_rl

**Comment by BY571** [comment]
@marioyc Thank you! I'll fix it :)
